files/image_config/platform/rc.local

```diff
@@ -183,8 +183,32 @@ for x in "$@"; do
 done
 }
 
+program_baud_rate()
+{
+    if [ -n "$aboot_platform" ]; then
+        platform=$aboot_platform
+    elif [ -n "$onie_platform" ]; then
+        platform=$onie_platform
+    else
+        platform=''
+    fi
+
+    # Default console speed: 9600
+    CONSOLE_SPEED=9600
+
+    CONSOLE_CFG=/usr/share/sonic/device/$platform/installer.conf
+    if [ -f $CONSOLE_CFG ]; then
```

Comment:
>-f [](start = 9, length = 2)

-r

Reply:
Using -r makes perfect sense.

However, I looked at our repo, I think we are using -f in this file and using -f more prevailingly in other files in our repo. I do see some '-r' tests though.

Maybe we could have a separate discussion and separate change to unify to one test?

Reply:
This code has been replaced with checking /proc/cmdline